x2gbfs/providers/moqo.py

```diff
@@ -182,8 +195,17 @@ def _extract_from_vehicles(
             },
         }
 
+        if vehicle.get('license'):
+            gbfs_vehicle['license_plate'] = re.split(r'[(|]', vehicle['license'])[0].strip()
+
         if vehicle.get('latest_parking') is not None and vehicle.get('latest_parking', {}).get('id') is not None:
```

Comment:
As the id is used twice, for the existence check and assignment, you could assign it to a variable,
`latest_parking = vehicle.get('latest_parking', {}).get('id')` will work in all three cases:  if the `latest_parking` does not exist, it's value is or set.

Reply:
This expression leads to an exception:

  File "/home/ipl/ipl46_flinkster/x2gbfs/providers/moqo.py", line 202, in _extract_from_vehicles
    latest_parking_id = vehicle.get('latest_parking', {}).get('id')
AttributeError: 'NoneType' object has no attribute 'get'

For this reason, I used two variables latest_parking and latest_parking_id

Reply:
Ok.